toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -383,6 +390,49 @@ func initializeRpmDatabase(installRoot string) (err error) {
 	return
 }
 
+// TdnfInstall installs a package in the current environment
+func TdnfInstall(packageName, installRoot string, currentPackagesInstalled, totalPackages int, reportProgress bool) (packagesInstalled int, err error) {
```

Comment:
I agree we should export a public `TdnfInstall`, but forcing all invokers to provide `currentPackagesInstalled`, `totalPackages`, and `reportProgress` seems cumbersome. There's only one specific instance where we want the progress reporting feature turned on, so I'd argue the default invocation of a `TdnfInstall` should have fewer arguments.

A `TdnfInstallNoProgress(....)` wrapper function here or something similair would simplify this.